NCS.DSS.Subscriptions/GetSubscriptionsByIdHttpTrigger/Function/GetSubscriptionsByIdHttpTrigger.cs

```diff
@@ -31,45 +17,45 @@ public GetSubscriptionsByIdHttpTrigger(IResourceHelper resourceHelper, IGetSubsc
             _httpResponseMessageHelper = httpResponseMessageHelper;
         }
 
-        [FunctionName("GetById")]
-        [ProducesResponseType(typeof(Models.Subscriptions), (int)HttpStatusCode.OK)]
-        [Response(HttpStatusCode = (int)HttpStatusCode.OK, Description = "Subscriptions found", ShowSchema = true)]
-        [Response(HttpStatusCode = (int)HttpStatusCode.NoContent, Description = "Subscriptions does not exist", ShowSchema = false)]
-        [Response(HttpStatusCode = (int)HttpStatusCode.BadRequest, Description = "Request was malformed", ShowSchema = false)]
-        [Response(HttpStatusCode = (int)HttpStatusCode.Unauthorized, Description = "API key is unknown or invalid", ShowSchema = false)]
-        [Response(HttpStatusCode = (int)HttpStatusCode.Forbidden, Description = "Insufficient access", ShowSchema = false)]
-        [Display(Name = "Get", Description = "Ability to retrieve a single subscriptions with a given SubscriptionsId for an individual customer.")]
-        [Disable]
-        public async Task<HttpResponseMessage> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "Customers/{customerId}/Subscriptions/{subscriptionId}")]HttpRequest req, ILogger log, string customerId, string subscriptionId)
-        {
-            log.LogInformation("C# HTTP trigger function processed a request.");
-
-            if (!Guid.TryParse(customerId, out var customerGuid))
-            {
-                log.LogWarning($"GetSubscriptionsByIdHttpTrigger customerId {customerId} BadRequest");
-                return _httpResponseMessageHelper.BadRequest(customerGuid);
-            }
-
-            if (!Guid.TryParse(subscriptionId, out var subscriptionsGuid))
-            {
-                log.LogWarning($"GetSubscriptionsByIdHttpTrigger subscriptionId {subscriptionId} BadRequest");
-                return _httpResponseMessageHelper.BadRequest(subscriptionsGuid);
-            } 
-
-            var doesCustomerExist = await _resourceHelper.DoesCustomerExist(customerGuid);
-
-            if (!doesCustomerExist)
-            {
-                log.LogWarning($"GetSubscriptionsByIdHttpTrigger customerGuid {customerGuid} NoContent");
-                return _httpResponseMessageHelper.NoContent(customerGuid);
-            }
-
-            var subscriptions = await _getSubscriptionsByIdService.GetSubscriptionsForCustomerAsync(customerGuid, subscriptionsGuid);
-            log.LogInformation($"GetSubscriptionsByIdHttpTrigger _getSubscriptionsByIdService customerGuid {customerGuid} subscriptionsGuid {subscriptionsGuid}");
-
-            return subscriptions == null ?
-                _httpResponseMessageHelper.NoContent(subscriptionsGuid) :
-                _httpResponseMessageHelper.Ok(JsonHelper.SerializeObject(subscriptions));
-        }
+        //[Function("GetById")]
```

Comment:
Please add this to the disabled functions document https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/4580999193/DSS+Disabled+Functions